submissions/AndriiBabushko/html-css_popup/index.html

```diff
@@ -0,0 +1,188 @@
+<!doctype html>
```

Comment:
Check your markup using [validator](https://validator.w3.org/)

Reply:
✓

Reply:
✓

Reply:
First time I thoght that I deleted it from a file, but I don't. It has alredy been fixed.